hedyorg/hedy
# 💻 variable list appears automatically open instead of hidden

Fixes #5147 

**How to test**

Follow these steps to verify this PR works as intended:

* Go to level 5 of Hedy (or another level in which variables exist)
* Create a program with at least two variables, for example
```
naam is Hedy
leeftijd is 15
print naam is leeftijd jaar oud
leeftijd is 40
print naam is leeftijd jaar oud
```
* Run the program step-by-step
* Check that the variable list and label button do not appear when there are no variables yet, but when there are, the variable list is shown without the need to click the label button. 
* Check that the variable list is hidden after you click the label button, and reappears if you click the label button again
* Check that the variable list stays hidden after clicking the label button, when continuing to run the program

**Checklist**
Done? Check if you have it all in place using this list: (mark with x if done)

- [ x] Contains one of the PR categories in the name
- [ x] Describes changes in the format above
- [ x] Links to an existing issue or discussion
- [ x] Has a "How to test" section

If you're unsure about any of these, don't hesitate to ask. We're here to help!
